src/components/drawer/drawer.scss

```diff
@@ -0,0 +1,23 @@
+@import "@material/drawer/mdc-drawer";
+@import "@material/list/mdc-list";
+@import "@material/button/mdc-button";
+
+.drawer {
+  height: 100%;
+  &__list-item {
+    position: relative;
+    &-more {
+      position: absolute !important;
```

Comment:
please -- DO NOT USE IMPORTANT!!! 

Yes, I understand that it won't work without it here, but the key thing to understand -- you should use important ONLY IN EXCEPTIONAL CASES.

How to fix it here (steps):

1 - revert webpack config to original state (without import fix for material components)
2 - remove imports from all of your scss files EXCEPT index.scss
3 - move all your imports to index.scss files, but you need to import *.css, not *.scss file from each package - so your index.scss will look like this

```scss
@import "~normalize.css";

@import "~@material/toolbar/dist/mdc.toolbar";
@import "~@material/button/dist/mdc.button";
@import "~@material/drawer/dist/mdc.drawer";
@import "~@material/list/dist/mdc.list";

@import "./components/variables";

@import "./components/page/page";
@import "./components/app/app";
@import "./components/drawer/drawer";
@import "./components/toolbar/toolbar";

```

notice that I'm importing from dist directory of the package  e.g. `@material/button/dist/...`

4 - you have a mixin in toolbar.scss - it will now won't compile -- so you need to remove mixin -- use this now
```scss
.toolbar {
  display: inline-flex;
  flex-direction: column;
  height: 100%;
  box-sizing: border-box;

  &__button,
  &__button:not(:disabled) {
    color: white;
  }
}

```
and it will work ok

5 - now if you remove importants all will work